src/routes/grantform/index.js

```diff
@@ -17,7 +17,13 @@ export default [
   {
     path: '/grant/success',
 
-    async action() {
+    async action({ store }) {
+      const { submission } = store.getState();
+
+      if (!submission) {
+        return { redirect: '/' };
+      }
```

Comment:
Yeah, there are plenty in there, you can log it out to see. My most used ones are 
```
{ store, userAgent, params }
```
```
path: '/admin/stuff/:id',
  async action({ store, userAgent, params }) {
    const { user } = store.getState();
    const { id } = params; //get the id from the url
...
```